tutorial/practicals/nemo/2_nemo_profiling/Makefile

```diff
@@ -41,39 +41,62 @@ include ../../../common.mk
 
 # We use the PSyData simple_timing implementation since it has no
 # dependencies.
-PROFILE_DIR ?= ../../../../lib/profiling/simple_timing
-PROFILE_LINK = -lsimple_timing
-PROFILE_LIB = ${PROFILE_DIR}/libsimple_timing.a
+PROFILE_WRAPPER_DIR ?= ../../../../lib/profiling/simple_timing
+PROFILE_WRAPPER_LIB ?= ${PROFILE_WRAPPER_DIR}/libsimple_timing.a
+PROFILE_WRAPPER_LINK ?= ${PROFILE_WRAPPER_LIB}
+# There's no separate library for 'simple_timing' - its functionality
+# is incorporated in the PSyData wrapper.
+PROFILE_DIR ?=
+PROFILE_LIB ?= 
+PROFILE_LINK ?=
+
+# For dl_timer
+# NOTE: dl_timer *requires* an initialisation call to be added
+#       to runner.f90:
+#         use profile_psy_data_mod, only : profile_PSyDataInit
+#         call profile_PSyDataInit()
+# (git clone https://bitbucket.org/apeg/dl_timer.git
+#  cd dl_timer; . compiler_setup/gnu_setup.sh; make dm_lib)
+#PROFILE_WRAPPER_DIR = ../../../../lib/profiling/dl_timer
+#PROFILE_WRAPPER_LIB = ${PROFILE_WRAPPER_DIR}/libdl_timer_psy.a
+#PROFILE_WRAPPER_LINK = ${PROFILE_WRAPPER_LIB}
+#PROFILE_DIR = ../../../../../dl_timer
+#PROFILE_LIB = ${PROFILE_DIR}/libdl_timer_mpi.a
+#PROFILE_LINK = ${PROFILE_LIB}
+# Since we've built the distributed-memory version of dl_timer
+# ('dm_lib') we need to ensure the executable is linked with MPI.
+#F90 = mpif90
```

Comment:
I tried to follow the instructions, and it was straight forward. It might be slightly better if the variables are set with `?=`, so you can just uncomment the lines, and then use `PROFILE_DIR=/my/path make`.

But: when I tried to run, I get:
```
TIMING: ERROR: timer_init() must be called after MPI_Init()!
```
I assume that's because there is no call to `MPI_Init` (once I added this, it was fine).

Wouldn't it be better to just use the non-mpi version of the timer library?

Reply:
That's weird. I thought I tried it and it worked. However, OMP would seem to be simpler so I'll swap to that.

Reply:
Done. I've checked it locally and it works as expected.

Reply:
Thanks